ckanext/dcatapchharvest/profiles.py

```diff
@@ -36,8 +36,6 @@
 EUTHEMES = dh.EUTHEMES
 ODRS = Namespace('http://schema.theodi.org/odrs#')
 
-GEOJSON_IMT = 'https://www.iana.org/assignments/media-types/application/vnd.geo+json'  # noqa
```

Comment:
I removed this because it's not used anywhere - seems like it was copied from profiles.py in ckanext-dcat but never needed.

Reply:
thanks